core/eth/contracts/FluxAggregator_test.go

```diff
@@ -249,3 +249,63 @@ func TestFluxAggregatorClient_LatestSubmission(t *testing.T) {
 		})
 	}
 }
+
+func TestFluxAggregatorClient_DecodesLogs(t *testing.T) {
+	fa, err := contracts.NewFluxAggregator(nil, common.Address{})
+	require.NoError(t, err)
+
+	newRoundLogRaw := cltest.LogFromFixture(t, "../../services/testdata/new_round_log.json")
+	var newRoundLog contracts.LogNewRound
+	err = fa.UnpackLog(&newRoundLog, "NewRound", newRoundLogRaw)
+	require.NoError(t, err)
+	require.Equal(t, int64(1), newRoundLog.RoundId.Int64())
+	require.Equal(t, common.HexToAddress("f17f52151ebef6c7334fad080c5704d77216b732"), newRoundLog.StartedBy)
+	require.Equal(t, int64(15), newRoundLog.StartedAt.Int64())
+
+	type BadLogNewRound struct {
```

Comment:
Why is it bad ?

Reply:
In Solidity, the round ID field is `roundId`.  Geth's Go code expects this to translate to `RoundId`, not `RoundID`, and fails to find the field if it doesn't.

Reply:
https://github.com/ethereum/go-ethereum/pull/16648 <- does go-eth support these 'abi' tags?

Reply:
Only on non-indexed fields.  I plan to rework their decoding logic in a subsequent PR to add this feature for indexed fields.

Reply:
Ok as long as it was considered

Reply:
Can we pls have a comment explaining this. Not obvious.